CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ add_subdirectory(DDCond)
 add_subdirectory(DDAlign)
 add_subdirectory(DDEve)
 add_subdirectory(DDDB)
+add_subdirectory(DDTest)
 
 dd4hep_enable_tests( DDTest )
```

Comment:
DDTest is added if BUILD_TESTING=ON, this is not the proper solution for your problem. There is an issue in UtilityApps/CMakeLists.txt where DDTest is required for one executable, but then things fail.